modules/apps/digital-signature/digital-signature-test/src/testFunctional/macros/DigitalSignature.macro

```diff
@@ -182,4 +222,10 @@ definition {
 			locator1 = "ListGroupItem#ITEM_TEXT");
 	}
 
+	macro recycleMessageSucess {
+		Click(locator1 = "Icon#MOVE_TO_RECYCLE_BIN");
+
+		VerifyElementPresent(locator1 = "Message#SUCCESS");
+	}
```

Comment:
Here you can change macro's name to: `deleteCreatedDM`, add these steps:
```
  DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
  PortletEntry.selectAll();
  DigitalSignature.clickCheckboxDM(checkboxDM = "_com_liferay_document_library_web_portlet_DLAdminPortlet_rowIdsFolder");
  DigitalSignature.recycleMessageSucess();
```
And add this macro on the tearDown due to all tests delete the DM created